src/bundle/ApiPlatform/OpenApiFactory.php

```diff
@@ -30,6 +32,7 @@ public function __construct(
     public function __invoke(array $context = []): OpenApi
     {
         $openApi = ($this->decorated)($context);
+        $openApi = $openApi->withInfo((new Info('Ibexa DXP REST API', Ibexa::VERSION, 'TODO: Description')));
```

Comment:
Description is not mandatory so it could be kept blank.

```suggestion
        $openApi = $openApi->withInfo((new Info('Ibexa DXP REST API', Ibexa::VERSION)));
```

The title should probably be a constant somewhere…